.gitignore

```diff
@@ -1,4 +1,10 @@
 openapitools.json
 node_modules/
 package-lock.json
-package.json
\ No newline at end of file
+package.json
+
+#Python
+python/
+# Environments
+*env*
+*ENV*
```

Comment:
remove line end

Reply:
It's opposite @ftoldato , GitHub is asking to add the line.

Reply:
@JoiceFelix if GitHub place the simbol ⊖ at the end of the file, it means that the file end of the file is wrong. So, keep one and just one line at the end of the file.